src/views/AddItem.jsx

```diff
@@ -12,17 +12,43 @@ export function AddItem({ listToken }) {
 		addItemToList();
 	};
 
+	// Take current list items and remove punctuation/spaces.
+	const removePunc = data.map((item) =>
+		item.name.toLowerCase().replace(/[^a-zA-Z0-9]/g, ''),
```

Comment:
This might be a big re-factor, but I wonder if we should handle this in its own function?

Right now it looks like we are partially validating the input in `handleNameInput` by accounting for multiple spaces, then in `removePunc` we are accounting for upper/lower case & numbers.

Maybe it would be more clean to make a `validateInput` function to get rid of excess spaces, remove any numbers or special characters, and make all letters lower case. We could then use styling to capitalize the first letter of each list item for the user.

Then we could then call this validation function both when we `setItemName` & when we `addItemToList` & that would keep the data in firebase really clean too?

Reply:
Agreed. Let's consolidate and create a `validateInput` function to clean up the item name. Then, we can use call`validateInput` in `addItemToList` function to validate the input before adding it to the Firebase database.

Reply:
We are looking into this. Will get back to you after we meet on Friday.

Reply:
Wow, really great job with these updates! It's looking so good!